velox/exec/tests/utils/AssertQueryBuilder.cpp

```diff
@@ -164,6 +164,15 @@ std::shared_ptr<Task> AssertQueryBuilder::assertResults(
   return cursor->task();
 }
 
+std::shared_ptr<Task> AssertQueryBuilder::assertNumRows(
```

Comment:
We can't assert the results, but can at least check that the types and number of rows returned are correct.